storage/tianmu/core/tianmu_attr_exqp.cpp

```diff
@@ -954,6 +959,16 @@ void TianmuAttr::EvaluatePack_BetweenInt(MIUpdatingIterator &mit, int dim, Descr
     } else
       EvaluatePack_NotNull(mit, dim);
   }
+
+#ifdef DEBUG_EVALUATE_PACK
+  auto diff =
+      std::chrono::duration_cast<std::chrono::duration<float>>(std::chrono::high_resolution_clock::now() - start);
```

Comment:
It's better to write these metrics to performance schema.

Reply:
That's a bad idea, because if you write monitors, you'll add extra runtime overhead and slow down query performance, especially on more low-level interfaces